### ISSUE TITLE
SignedURL using pem for PrivateKey not successful

### ISSUE THREAD

**Issue by josephbergevin** [GoogleCloudPlatform/gcloud-golang#109]
package: `storage`

I experimented with using a `pem` file and a `rsa` file to produce my private key. `rsa` worked, `pem` did not. 
Both compiled and gave no errors to be logged. The error came when trying to access the URL with a key generated from the `pem` version of the `p12` key.

```Error Code: SignatureDoesNotMatch
Message: The request signature we calculated does not match the signature you provided. Check your Google secret key and signing method.```

It's fine if rsa is the way to do it. If so, maybe just change the instructions given in storage.go for the `PrivateKey` field in the `SignedURLOptions` struct.

**Comment by gmlewis** [comment]
That's odd... PEM files have worked fine for me.

**Comment by josephbergevin** [comment]
Strange. Yeah, I tried loading the pem from a file and from a string and had no luck. Then I perused the test code in storage_test.go and saw that a rsa file was being used as well for the key. When I converted the p12 to rsa and used that instead, it worked.
I think this issue could be resolved simply by adding to the instructions on the `PrivateKey` field. Maybe give additional instructions for converting the `p12` to `rsa`.
